### ISSUE TITLE
ENH: add asv's which include EAs where appropriate

### ISSUE THREAD

**Issue by jreback** [pandas-dev/pandas#24351]
specially talking about the IntNA EA's can be added in lots of places for perf testing.

**Comment by Lifreed** [comment]
I see tag "good first issue", that's what I need. I can try solve this problem but I need more information.

**Comment by Lifreed** [comment]
I see tag "good first issue", that's what I need. I wont solve this problem but I need more information.

**Comment by jselvam11** [comment]
Hi, I am interested in contributing to this project. Would it be fine if I could work on this project?

**Comment by rotuna** [comment]
@jreback 

Do you want more tests which use intNA instead of np.nan? For example, time_groupby_nth_all from group.py in asv tests uses np.nan. Adding extra tests could show how much faster/slower intNA make things. 

Is this what you were gunning for or should it be something else entirely?

**Comment by jreback** [comment]
yeah ideally we would like to test intNA dtypes alongside existing ones

**Comment by rotuna** [comment]
I found np.nan's in the following files

- [ ] algorithms.py
- [ ] csv.py
- [ ] sql.py
- [ ] frame_methods.py
- [ ] sparse.py 
- [ ] categoricals.py
- [ ] reindex.py 
- [ ] groupby.py
- [ ] strings.py 
- [ ] series_methods.py 
- [ ] timeseries.py 
- [ ] replace.py

**Comment by shuaggar-sys** [comment]
@jreback Is this closed?

**Comment by jreback** [comment]
you would have to look and see where it's missing

**Comment by jbrockmendel** [comment]
i think we've done a decent enough job of this that the issue is no longer needed.  closing.
